pgjdbc/src/test/java/org/postgresql/test/jdbc4/ArrayTest.java

```diff
@@ -295,27 +313,34 @@ public void testUUIDArray() throws SQLException {
   @Test
   public void testSetObjectFromJavaArray() throws SQLException {
     String[] strArray = new String[]{"a", "b", "c"};
+    Object[] objCopy = new Object[strArray.length];
+    System.arraycopy(strArray, 0, objCopy, 0, strArray.length);
```

Comment:
Done.
Out of curiosity, why do you prefer the Arrays wrapper? These are functionally equivalent things, skipping some of the reflection stuff that Arrays has to do (or in this specific case because it is `Object[]`, just a class identity comparison).